.github/workflows/Test.yml

```diff
@@ -483,7 +483,9 @@ jobs:
         run: pip3 install -r testsuite/requirements.txt
 
       - name: '🚦 Test installation of pyGHDL through pip'
-        run: pip install ("git+https://github.com/ghdl/ghdl.git@" + $env:GHDL_HASH)
+        run: |
+          pip install pyTooling wheel
```

Comment:
This should not be required, since we added section `build-system` to `pyproject.toml`?

Reply:
Fixed with latest push.